### PR TITLE
Rm Geometry, mv Mesh to own file

### PR DESCRIPTION
**Pull request by zzmp**

### CODE REVIEW

**Comment by zzmp** [comment]
#### Testing

General smoke screen.

File: plugins/openvr/src/ViveControllerManager.h [hunk 1]
```diff
@@ -100,7 +99,6 @@ class ViveControllerManager : public InputPlugin {
 
     bool _registeredWithInputMapper { false };
     bool _modelLoaded { false };
-    model::Geometry _modelGeometry;
```
Comment (Atlante45) [inline plugins/openvr/src/ViveControllerManager.h]: This is used in some commented out code, can you remove it?

**Comment by hifi-gustavo** [comment]
Build console output for commit: f785be2e40cb60198425752a79609e0d7230cf48 available at: https://gist.github.com/064a8a2dfc5506e87f74<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7442.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7442.dmg

**Comment by Atlante45** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: f785be2e40cb60198425752a79609e0d7230cf48 available at: https://gist.github.com/efb6be38427a4a2adcdd<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7442.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7442.dmg

**Comment by samcake** [comment]
I would say this
THis is not necessary

**Comment by ZappoMan** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: f785be2e40cb60198425752a79609e0d7230cf48 available at: https://gist.github.com/efb6be38427a4a2adcdd<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7442.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7442.dmg

**Comment by ZappoMan** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: f785be2e40cb60198425752a79609e0d7230cf48 available at: https://gist.github.com/efb6be38427a4a2adcdd<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7442.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7442.dmg

**Comment by thoys** [comment]
QA test: Meshes still load as expected. :+1: :heavy_check_mark:

File: plugins/openvr/src/ViveControllerManager.h (hunk 1 above)
Comment (AndrewMeadows) [inline plugins/openvr/src/ViveControllerManager.h]: The commented out code is in plugins/openvr/src/ViveControllerManager.cpp

**Pull request closed without merging**